frontend/opensignal-web/src/store/index.tsx

```diff
@@ -15,19 +15,19 @@ type State = {
     tokenBalance: number;
     tokenBalanceTrigger: boolean;
 };
-type GitcoinProviderProps = {children: React.ReactNode};
-let myweb3: any = new Web3(window.ethereum);
+type GitcoinProviderProps = { children: React.ReactNode };
+const myweb3: any = new Web3(window.ethereum);
 
-const OPENSIGNAL_LABEL = 'OpenSignal';
-const TOKEN_LABEL = 'OpenSignalToken';
+const OPENSIGNAL_LABEL = "OpenSignal";
+const TOKEN_LABEL = "OpenSignalToken";
 const openSignalContract =
     localStorage.getItem(OPENSIGNAL_LABEL) ||
-    'ipfs://bafyreibraij4zlc7wt52duyr5enkmglpooh4iimanh4ftxs2gar55z24re/metadata.json';
+    "ipfs://bafyreibraij4zlc7wt52duyr5enkmglpooh4iimanh4ftxs2gar55z24re/metadata.json";
```

Comment:
I think that to improve the developer experience we can start with an environment variable.
And then we can do something like an import of the ABIs from the contracts package.